aptos-move/aptos-aggregator/src/types.rs

```diff
@@ -1,7 +1,9 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::bounded_math::code_invariant_error;
```

Comment:
Note for the future: the errors are quite ugly right now, which is highlighted by the fact that we have code invariant error in bounded math😂 I added `AggregatorError` to solve that problem, hopefully we can switch over to that. It's hard to do it with so many PRs in flight concurrently though